flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/ATSU/A320_Neo_CDU_AOC_OFPData.js

```diff
@@ -504,59 +365,59 @@ async function loadFuel(mcdu, updateView) {
     updateView();
 }
 
+const payloadConstruct = new A32NX_PayloadConstructor();
+const paxStations = payloadConstruct.paxStations;
+const cargoStations = payloadConstruct.payloadStations;
+
+const MAX_SEAT_AVAILABLE = 174;
+const PAX_WEIGHT = 84;
+const BAG_WEIGHT = 20;
+
 /**
- * Calculate the baggage payload distribution to achieve target CG
- * @param {number} payload in pound
- * @param {number} percentMacTarget Target %MAC CG / 100
- */
-function getDistributedBaggagePayload(payload = 0, percentMacTarget = 30) {
-    const leMacZ = -5.233333; // Value from Debug Weight
-    const macSize = 14.0623; // Value from Debug Aircraft Sim Tunning
-    const EMPTY_AIRCRAFT_WEIGHT = 90400 * 0.453592; // Value from flight_model.cfg to kgs
-    const EMPTY_AIRCRAFT_POSITION = -9; // Value from flight_model.cfg
-    const FORWARD_BAGGAGE_POSITION = 21.825772; // Value from flight_model.cfg
-    const REAR_BAGGAGE_POSITION = -35.825745; // Value from flight_model.cfg
+     * Calculate %MAC ZWFCG of all stations
+     */
+function getZfwcg() {
+    const currentPaxWeight = PAX_WEIGHT + BAG_WEIGHT;
 
-    const cgTarget = leMacZ - (macSize * (percentMacTarget / 100));
+    const leMacZ = -5.39; // Value from Debug Weight
```

Comment:
I also just noticed that these leMacZ and macSize values are not the same values I gave you. They have been rounded off like they are in the dev mode aircraft editor debug weight window. This will not result in the same cg value that MSFS calculates and shows in the weight and balance UI window. (I just did some sample cg calcs for a cg calculation spreadsheet and found this.) These values should be the ones I gave you before:  leMacZ = -5.386 and macSize = 13.454.  Thanks!

Reply:
That's right. I remember you sent the values without rounding, and I also remember changing this myself. Must've forgot to commit with the other changes. Thank you for taking a second look, will fix this one too 👍🏻